### DISCUSSION TITLE
Milvus-backup support another MinIO service

### DISCUSSION THREAD

**Discussion by feelever** [milvus-io/milvus#26318] [Q&A and General discussion]
When we review the config and mivus-backup tool，it only have one StorageClient to transfer the source to backup; However,in some case，we want to backup the data into another MinIO or object storage service, make sure to avoid MinIO crash or k8s node issue break down our service; now we do it by explose the port and backup throungh MinIO API and store it into our object storage service;

**Comment by xiaofan-luan** [comment]
I think it's actually very reasonable. 
Anyone want to contribute on it?
